# Generated by Django 3.0.4 on 2020-03-18 11:37

import datetime
from django.db import migrations, models
from django.utils.timezone import utc


class Migration(migrations.Migration):

    dependencies = [
        ('resume', '0002_auto_20200318_0636'),
    ]

    operations = [
        migrations.AlterField(
            model_name='medical_data',
            name='BP_STBL',
            field=models.IntegerField(),
        ),
        migrations.AlterField(
            model_name='medical_data',
            name='COMFORT',
            field=models.IntegerField(),
        ),
        migrations.AlterField(
            model_name='medical_data',
            name='CORE_STBL',
            field=models.IntegerField(),
        ),
        migrations.AlterField(
            model_name='medical_data',
            name='DESCISION',
            field=models.IntegerField(),
        ),
        migrations.AlterField(
            model_name='medical_data',
            name='L_BP',
            field=models.IntegerField(),
        ),
        migrations.AlterField(
            model_name='medical_data',
            name='L_CORE',
            field=models.IntegerField(),
        ),
        migrations.AlterField(
            model_name='medical_data',
            name='L_O2',
            field=models.IntegerField(),
        ),
        migrations.AlterField(
            model_name='medical_data',
            name='L_SURF',
            field=models.IntegerField(),
        ),
        migrations.AlterField(
            model_name='medical_data',
            name='SURF_STBL',
            field=models.IntegerField(),
        ),
        migrations.AlterField(
            model_name='registration_data',
            name='pub_date',
            field=models.DateField(default=datetime.datetime(2020, 3, 18, 11, 37, 42, 912428, tzinfo=utc)),
        ),
    ]
